poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.4.0 and should not be changed by hand.
```

Comment:
Please revert any changes not connected with the removal of `cachy`.

Reply:
Since removing it from pyproject.toml, the lock file needs to be regenerated, no?

Reply:
yes, but as you can see, you not only removed cachy, but also updated other dependencies, which shouldn't be the case (not as a part of this PR at least)

Reply:
Ahh. Did `poetry lock --no-update` and fixed. Thanks.